policyengine_us/variables/gov/usda/snap/dc_snap_temporary_local_benefit.py

```diff
@@ -0,0 +1,29 @@
+from policyengine_us.model_api import *
+
+
+class dc_snap_temporary_local_benefit(Variable):
+    value_type = float
+    entity = SPMUnit
+    definition_period = MONTH
```

Comment:
is this monthly or yearly? If monthly then we need to adjust the months in year logic